share/completions/distrobox.fish

```diff
@@ -0,0 +1,115 @@
+set -l commands assemble create rm enter list ls stop upgrade ephemeral generate-entry version help
+
+complete -c distrobox -f
+
+function __fish_distrobox_list_containers
+    distrobox ls | tr -d ' ' | cut -d "|" -f2 | tail -n +2
+end
+
+complete -c distrobox \
+    -s V -l version -d "Show version"
+complete -c distrobox -n "__fish_seen_subcommand_from $commands" \
+    -s v -l verbose -d "Show more verbosity"
+complete -c distrobox -n "__fish_seen_subcommand_from $commands" \
+    -s h -l help -d "Show help message"
+complete -c distrobox -n "__fish_seen_subcommand_from enter ephemeral list ls create rm generate-entry upgrade && not __fish_seen_subcommand_from assemble" \
+    -s r -l root -d "Launch podman/docker/lilipod with root privileges"
+
+complete -c distrobox --no-files -n "not __fish_seen_subcommand_from $commands" \
```

Comment:
first you use -f then the long variant (--no-files);
let's be consistent, and use only the short variant like most scripts